.github/workflows/maven.yml

```diff
@@ -103,3 +103,12 @@ jobs:
         with:
           name: logs-${{ runner.os }}
           path: '**/target/**/*.log'
+  event_file:
+    name: "Upload Event File"
+    runs-on: ubuntu-latest
+    steps:
+    - name: Upload
+      uses: actions/upload-artifact@834a144ee995460fba8ed112a2fc961b36a5ec5a # v4.3.6
```

Comment:
is there a reason to use an exact tag?

Reply:
In general that's suggested practice to harden supply-chain security.
Otherwise it would be possible to publish a new artifact under a new tag (which is desired if one just uses the major version).
I can change it to just use `actions/upload-artifact@v4`. Without dependabot being setup it is otherwise tedious to update.

Reply:
Btw. since you asked in xtext-reference projects, one can configure dependabot to just update workflows:
https://github.com/eclipse-m2e/m2e-core/blob/master/.github/dependabot.yml

And it can also handle these specific commit-ids.